patches/minecraft/net/minecraft/world/level/block/FlowerBlock.java.patch

```diff
@@ -0,0 +1,40 @@
+--- a/net/minecraft/world/level/block/FlowerBlock.java
++++ b/net/minecraft/world/level/block/FlowerBlock.java
+@@ -15,9 +_,20 @@
+    private final MobEffect f_53508_;
+    private final int f_53509_;
+ 
++   private final java.util.function.Supplier<MobEffect> suspiciousStewEffectSupplier;
++
++   public FlowerBlock(java.util.function.Supplier<MobEffect> suspiciousStewEffectSupplier, int effectDuration, BlockBehaviour.Properties properties) {
++      super(properties);
++      this.f_53508_ = null;
++      this.suspiciousStewEffectSupplier = suspiciousStewEffectSupplier;
++      this.f_53509_ = effectDuration;
++   }
++
++   @Deprecated
+    public FlowerBlock(MobEffect p_53512_, int p_53513_, BlockBehaviour.Properties p_53514_) {
+       super(p_53514_);
+       this.f_53508_ = p_53512_;
++      this.suspiciousStewEffectSupplier = null;
```

Comment:
Move this at the start or end of constructor, no need to throw it in the middle.